src/library/Server/Manager/Plesk.php

```diff
@@ -189,9 +189,9 @@ public function changeAccountPassword(Server_Account $a, $new)
     	$this->getLog()->info('Changing password for account ' . $a->getUsername());
 
     	if ($a->getReseller()) {
-            $result = $this->_client->reseller()->setProperties('login', $a->getUsername(), ['passwd' => $a->getPassword()]);
+            $result = $this->_client->reseller()->setProperties('login', $a->getUsername(), ['passwd' => htmlspecialchars($new)]);
```

Comment:
I have tested a fresh Plesk 18 install with the latest FOSSBilling (preview), and can confirm `htmlspecialchars` is not needed. If you are getting an error without this - could you please advise how to reproduce? Otherwise, please update to just pass `$new`.

Reply:
@admdly thanks for testing. it was probably a caching problem, that i got about XML problem. Now after php-fpm restart, i don't get that error. Plesk Version = 
 <meta name="plesk-build" content="1800230907.05">

Reply:
Thanks for re-testing and making those changes. 🙂